service/configurations/client.go

```diff
@@ -63,3 +63,20 @@ func (c *Client) Get(ctx context.Context, account, name string) (*ConfigurationR
 	}
 	return configuration, nil
 }
+
+// Create a configuration on Upbound
+// Note that when using GitHub, this will make a new repo, and that requires
+// the GitHub app to be authorized and installed. That's a separate API, which
+// begins with the gitsources login. The full login is implemented in the CLI.
```

Comment:
```suggestion
// begins with the gitsources login. The full login is implemented in the up CLI (https://github.com/upbound/up).
```
nit: since the SDK is open source I think being explicit here would be good